src/main/java/org/wso2/carbon/esb/connector/utils/SessionManager.java

```diff
@@ -59,61 +49,55 @@ private String getKey(String host, int port, String systemId) {
      * @throws IOException
      */
     public SMPPSession getSmppSession(int enquireLinkTimer, int transactionTimer, String host, int port,
-                                      BindParameter bindParameter, int retryCount) throws IOException {
-        SMPPSession smppSession = smppSessionList.get(getKey(host, port, bindParameter.getSystemId()));
+                                      BindParameter bindParameter, int retryCount, String sessionName) throws IOException {
```

Comment:
This seems to exceed maximum line length. Shall we format it?

Reply:
This is fixed with https://github.com/wso2-extensions/esb-connector-smpp/pull/27/commits/3cf3826525c6a63a787dd40ed3a49541d720adfa